app/views/layout.mustache

```diff
@@ -120,5 +120,17 @@
   <a class="link github_issue" href="https://github.com/dart-lang/pub-dartlang-dart/issues/new">Report an issue</a>
 </footer>
 
+<script type="application/ld+json">
```

Comment:
squash this down a bit? Remove indent at least – since it's on every page.

Reply:
updated: generating JSON in templates.dart, compact, yet keeping it readable in the source

Reply:
Nice!